### DISCUSSION TITLE
preview feature

### DISCUSSION THREAD

**Discussion by abhicodes369** [otahina/PowerPoint-Generator-Python-Project#7] [Ideas]
hey @otahina nice project 👏👏 . we can add a preview feature to preview the ppt before downloading it

**Comment by otahina** [comment]
Hey, Thanks for your kind words and the thoughtful suggestion!  🙌 A preview feature sounds great! Feel free to fork and submit a Pull Request, or I can create an issue to discuss further. @abhicodes369 
Hina
